utils/docker/Dockerfile.centos.7

```diff
@@ -134,6 +134,7 @@ RUN dnf -y install deltarpm && \
         sg3_utils \
         sudo \
         valgrind-devel \
+        tbb-devel \
```

Comment:
I think we are aiming to keep these in alpha order so this should go above `valgrind-devel`.

I won't repeat this comment for all other Dockerfiles but please fix them where applicable.

Reply:
done